jest.config.js

```diff
@@ -32,10 +32,18 @@ const customJestConfig = {
   collectCoverageFrom: [
     'src/**/*.{js,jsx,ts,tsx}',
     '!src/**/*.stories.{js,jsx,ts,tsx}',
+    '!src/lib/drupal/query.ts',
```

Comment:
i think most of these in src/lib we will actually want to have tests for, no?

i'd also move these two down with the other src exclusions, maybe with a comment that we are excluding them for now, but they should be un-excluded as tests are written for them.

Reply:
I felt like a lot of these /src/lib file functions were based primarily around data fetching and more suitable for e2e or integration tests?  I can give them a look again if you feel they should be included for unit testing

Otherwise I'll comment and move

Reply:
i think they should have unit tests if at all possible. doesn't need to happen in this PR, there's a ton of good work here and i'd like to get it merged sooner than later